### PR TITLE
PTNFLY-1333 Migrate all "Content Views" Content Over to the Site.

### PR DESCRIPTION
**Pull request by cardosogabriel**

### CODE REVIEW

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-1.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-1.html]: We don't want to check this file into the widgets area. The widgets directory is only a soft link to the version of patternfly installed by bower.

This file should only be pushed to the patternfly repo.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-2.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-2.html]: This file should be added to the patternfly repo, instead.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-3.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-3.html]: This file should be added to the patternfly repo, instead.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-4.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-4.html]: This file should be added to the patternfly repo, instead.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-5.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-5.html]: This file should be added to the patternfly repo, instead.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-6.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
+<div class="list-group list-view-pf">
```
Comment (dlabrecq) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-6.html]: This file should be added to the patternfly repo, instead.

File: source/pattern-library/content-views/list-view/index.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+              <div class="container-fluid container-cards-pf">
+                <div class="row row-cards-pf">
+                  {% include widgets/list-view/list-view-variation-4.html %}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="card-markup-5" href="#card-markup-5">Reference Markup</a></p>
+        <div class="collapse in" id="card-markup-5">
```
Comment (dlabrecq) [inline source/pattern-library/content-views/list-view/index.md]: Just a nit. Remove the "card-" prefix from "card-markup-5" and others -- there's no cards here. This id seems to be copied from other pages.

**Comment by dlabrecq** [comment]
Pulled the code, built and viewed in my browser. Everything looks good to me, except for the files committed under the source/components directory.

File: source/components/patternfly/tests-src/_includes/widgets/list-view-variation-1.html (hunk 1 above)
Comment (cardosogabriel) [inline source/components/patternfly/tests-src/_includes/widgets/list-view-variation-1.html]: Update the PR removing the list-view-variation htmls that u indicated.

File: source/pattern-library/content-views/list-view/index.md (hunk 7 above)
Comment (cardosogabriel) [inline source/pattern-library/content-views/list-view/index.md]: Replaced "card-markup-n" by "markup-n", with n from 1 to 7.

**Comment by cardosogabriel** [comment]
Hope now it is fine!

**Comment by dlabrecq** [comment]
LGTM

**Comment by dlabrecq** [comment]
Just need to pull the latest and resolve the merge conflict.

**Comment by cardosogabriel** [comment]
Should be good now ;)

**Pull request merged**